docs/src/main/sphinx/connector/sqlserver.rst

```diff
@@ -119,26 +119,186 @@ that catalog name instead of ``sqlserver`` in the above examples.
 Type mapping
 ------------
 
-Trino supports the following SQL Server data types:
-
-==================================  ===============================
-SQL Server Type                     Trino Type
-==================================  ===============================
-``bigint``                          ``bigint``
-``tinyint``                         ``smallint``
-``smallint``                        ``smallint``
-``int``                             ``integer``
-``float``                           ``double``
-``char(n)``                         ``char(n)``
-``varchar(n)``                      ``varchar(n)``
-``date``                            ``date``
-``datetime2(n)``                    ``timestamp(n)``
-``datetimeoffset(n)``               ``timestamp(n) with time zone``
-==================================  ===============================
+Because Trino and SQL Server each support types that the other does not, this
+connector modifies some types when reading or writing data.
```

Comment:
Would suggest adding a sentence clarifying that the type mappings are not bi-directional. Something like "Data types may not map the same way in both directions between Trino and the data source. Refer to the following sections for type mapping in each direction." or similar

Reply:
Good suggestion!